packages/devtools_app/lib/src/memory/memory_screen.dart

```diff
@@ -167,6 +175,76 @@ class MemoryBodyState extends State<MemoryBody>
       });
     });
 
+    addAutoDisposeListener(eventChartController.tapNotifier, () {
+      if (!eventChartController.tapNotifier.value.isEmpty &&
```

Comment:
is there a reason we add these isEmpty checks rather than checking whether the value is null?

Reply:
isEmpty can be replaced with null and the correct null checks and ?. 
Done.